pom.xml

```diff
@@ -109,7 +115,18 @@
                 <groupId>jakarta.xml.ws</groupId>
                 <artifactId>jakarta.xml.ws-api</artifactId>
                 <version>${jakarta.xml.ws-api.version}</version>
-            </dependency>     
+            </dependency>
+            <dependency>
+                <groupId>org.glassfish.jaxb</groupId>
+                <artifactId>jaxb-runtime</artifactId>
```

Comment:
There is also `quarkus-jaxb`... 🤔

Reply:
Good point @famod we already have a dependency on quarkus-jaxb, which has everything we need; I have done some additional cleanup.